2023-02-cloud-octobot-plans.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Introducing Cloud OctoBot plans
+subtitle: Get your the perfect Cloud OctoBot
+slug: introducing-cloud-octobot-plans
+tags: hosting, cryptocurrency, trading, strategy, exchange, blog, octobot cloud
+cover: https://raw.githubusercontent.com/Drakkar-Software/OctoBot-Blog/master/resources/images/octobots-in-octobot-cloud/cover.jpg
+domain: blog.octobot.online
+--- 
+
+
+# An OctoBot plan for each need
+
+In the OctoBot team, we want everyone to be able to use OctoBot and enjoy great trading strategies.  
+
+Our goal is that whether you want to get a cheap and pre-configured strategy or infinite customization to create your very own strategy, you should be able to do it using OctoBot.
+
+That's why we created 4 different plans that are designed to fit every user. 
+
+# The plans
+
+## Starter
+> Comming soon
```

Comment:
I think we will make it available in this update. I will still have the web interface.